File: MODUL 2/PYTHON/PRAK205-2310817320006-Alya_Rosaan.py
import math
#case 1
def mynum1(a,b):
    alas = math.sqrt(pow(b, 2)-pow(a, 2))
    return alas
def mynum2(a,b):
    tinggi = a
    return tinggi
def mynum3(a,b):
    alas = math.sqrt(pow(b, 2)-pow(a, 2))
    keliling = a + b + alas
    return keliling
def mynum4(a,b):
    alas = math.sqrt(pow(b, 2)-pow(a, 2))
    luas = 1/2*alas*a
    return luas

print("input")
a,b= map(int, input().split())

hasil= mynum1(a,b)
print("alas", int(hasil), "cm")

hasil_1= mynum2(a,b)
print("tinggi", int(hasil_1), "cm")

hasil_2= mynum3(a,b)
print("keliling", int(hasil_2), "cm")

hasil_3= mynum4(a,b)
print("luas", int(hasil_3), "cm^2")

#case 2

print("input")
a = int(input())
b = int(input())

hasil= mynum1(a,b)
print("alas", int(hasil), "cm")

hasil_1= mynum2(a,b)
print("tinggi", int(hasil_1), "cm")

hasil_2= mynum3(a,b)
print("keliling", int(hasil_2), "cm")

hasil_3= mynum4(a,b)
print("luas", int(hasil_3), "cm^2")

File: MODUL 2/C/PRAK205-2310817320006-Alya_Rosaan.c
#include <stdio.h>
#include <math.h>

int main(){

    int a,b;
    int c,d;

    printf("input\n");
    printf("");
    scanf("%d %d", &a, &b);

    int alas = sqrt(pow(b, 2)-pow(a, 2));
    int tinggi = a;
    int keliling = a + b + alas;
    int luas = alas*tinggi/2;

    printf("output\n");
    printf("alas = %d cm\n", alas);
    printf("tinggi = %d cm\n", tinggi);
    printf("keliling = %d cm\n", keliling);
    printf("luas = %d cm^2\n\n", luas);

    printf("input\n");
    printf("");
    scanf("%d", &c);
    printf("");
    scanf("%d", &d);

    int alas1 = sqrt(pow(d, 2)-pow(c, 2));
    int tinggi1 = c;
    int keliling1 = c + d + alas1;
    int luas1 = alas1*tinggi1/2;

    printf("output\n");
    printf("alas = %d cm\n", alas1);
    printf("tinggi = %d cm\n", tinggi1);
    printf("keliling = %d cm\n", keliling1);
    printf("luas = %d cm\n\n", luas1);

}

